test/next/src/app/vercel/blob/app/test/client-serverless/page.tsx

```diff
@@ -0,0 +1,44 @@
+'use client';
+import { put, type BlobResult } from '@vercel/blob';
+import { useEffect, useState } from 'react';
+
+export default function AppBodyClient({
+  searchParams,
+}: {
+  searchParams: { filename: string };
```

Comment:
Is this for our automated test to pass a filename before hand? I am not following why we're reading filename from query params yet

Reply:
it's used as test identifier (same for all the other file upload tests)